schema/defs-linux.json

```diff
@@ -203,7 +204,8 @@
                     "$ref": "#/definitions/Minor"
                 },
                 "access": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^[rwm]{1,3}$"
```

Comment:
In my opinion ,`^(?!.*(.).*\1)[rwm]{1,3}$` will work well, but it seems json schema does not support `?!`.